### PR TITLE
[New] Add js-objectdetect@1.0.0 w/ git auto-update

### LINKED ISSUE #10269: [Request] Add js-objectdetect
**Library name:** js-objectdetect
**Git repository url:** https://github.com/mtschirs/js-objectdetect
**License(s):** MIT License
**Official homepage:** https://github.com/mtschirs/js-objectdetect

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #10269, cc @mtschirs, using web-minify-helper for minifing files

Pull request for issue: #10269
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mtschirs/js-objectdetect
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 87
   - Count of stars: 1092
   - Count of forks: 237
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/js-objectdetect/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "object detection"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mtschirs/js-objectdetect.git",
+    "fileMap": [
+      {
+        "basePath": "js",
+        "files": [
+          "objectdetect*"
```
Comment (sufuf3) [inline ajax/libs/js-objectdetect/package.json]: We don't use `js` as the basePath.
Please keep `js` folder.
Thanks

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

File: ajax/libs/js-objectdetect/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "js-objectdetect",
+  "filename": "objectdetect.min.js",
```
Comment (extend1994) [inline ajax/libs/js-objectdetect/package.json]: `js/objectdetect.min.js` related to `basePath`

File: ajax/libs/js-objectdetect/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mtschirs/js-objectdetect.git"
+  },
+  "description": "computer vision in your browser - javascript real-time object detection",
+  "keywords": [
+    "computer vision",
+    "object detection"
```
Comment (extend1994) [inline ajax/libs/js-objectdetect/package.json]: (Discussion not request)
Not sure if it's a better idea to separate them into
```js
"keywords": [
    "computer",
    "vision",
    "object",
    "detection"
]
```

File: ajax/libs/js-objectdetect/package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "js-objectdetect",
```
Comment (extend1994) [inline ajax/libs/js-objectdetect/package.json]: Missing `version: "1.0.0"`

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

**Pull request merged**